ecommerce/src/main/java/app/config/WebSecurityConfig.java

```diff
@@ -24,7 +24,7 @@ public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception
 
 	}
 
-	private static final String[] AUTH_WHITELIST = { "/", "/login", "/logout", "/assets/**", "/products/**",
+	private static final String[] AUTH_WHITELIST = { "/", "/login", "/logout", "/signup", "/assets/**", "/products/**",
 			"/blogs/**", "/shop-grid" };
```

Comment:
Sao ko để 
```suggestion
	private static final String[] AUTH_WHITELIST = { "/**" };
```
thử ri dc ko các bạn, chứ request nào cũng thêm vào thì chóng mặt lắm

Reply:
Em nghĩ để như chị noái thì mình k phân ra được admin với user á